src/service/extension-location.js

```diff
@@ -49,13 +49,19 @@ export function calculateExtensionScriptUrl(location, extensionId, isLocalDev) {
  * @param {string} entryPoint
  * @param {boolean=} isLocalDev
  * @param {boolean=} opt_rtv
+ * @param {boolean=} opt_minified
  * @return {string}
  */
 export function calculateEntryPointScriptUrl(
-    location, entryPoint, isLocalDev, opt_rtv) {
+    location, entryPoint, isLocalDev, opt_rtv, opt_minified) {
   const base = calculateScriptBaseUrl(location, isLocalDev);
-  if (opt_rtv) {
+  if (opt_rtv && opt_minified) {
```

Comment:
[This is what I had before.](https://github.com/ampproject/amphtml/pull/9475/commits/3669b49077f4f27756ebb55c9fbfadacd5f6d414#r118016939) @choumx @jridgewell which way do we want to go with?